src/apis/slackBot/queryFn.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+
+import { UserDBProps } from "@/hooks/api/useUserListByDB.ts";
+
+interface Props {
+  mentionedList: UserDBProps[];
+}
+export const postMessageSlackBot = ({ mentionedList }: Props) => {
+  const memberList = [] as UserDBProps[];
```

Comment:
저는 이렇게 쓰는 편인데 취향차인가요?
```suggestion
  const memberList: UserDBProps[] = [];
```

Reply:
[윤서님](https://github.com/prgrms-fe-devcourse/FEDC5_DevNamu_eunsu/pull/194/#discussion_r1452909007)과 통하셨군요!